modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/SnapshotManagerComponent.java

```diff
@@ -158,11 +156,16 @@ public DistributionSnapshot getSnapshot(String key, CoreSession session) {
         return snaps.get(0);
     }
 
+    protected Comparator<DistributionSnapshotDesc> getDistributionComparator() {
+        return Comparator.comparing(DistributionSnapshotDesc::getVersion, new VersionComparator())
+                         .reversed()
+                         .thenComparing(DistributionSnapshotDesc::getName);
```

Comment:
This should probably be stored in a static constant to avoid recomputing it each time we do a sort.